samples/javascript_nodejs/02.echobot-with-counter/index.js

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const path = require('path');
+const restify = require('restify');
+// Import required bot services. See https://aka.ms/bot-services to learn more about the different parts of a bot.
+const { BotFrameworkAdapter, ConversationState, MemoryStorage } = require('botbuilder');
+// Import required bot configuration.
+const { BotConfiguration } = require('botframework-config');
+// This bot's main dialog.
+const { Bot } = require('./bot');
```

Comment:
bot.js exports looks like this:  exports.EchoBot = EchoBot;

so wouldn't that make your require look like:  const { EchoBot } = require('./bot');